--- acm-5.0/V/lib/Vlib.h.tim	Wed Jul  7 14:53:45 1999
+++ acm-5.0/V/lib/Vlib.h	Wed Jul  7 14:54:08 1999
@@ -316,9 +316,6 @@
 	extern Status AlibAllocColor(Viewport *v, Colormap colormap, XColor *c);
 
 
-#ifndef SYSV
-	extern char *strdup(const char *);
-#endif
 
 #if (defined(_AIX) && defined(_IBMR2)) && !(defined(__GNUC__) \
      && (__GNUC__ == 2 ))
--- acm-5.0/V/lib/InitAWin.c.tim	Wed Jul  7 14:53:52 1999
+++ acm-5.0/V/lib/InitAWin.c	Wed Jul  7 14:54:01 1999
@@ -6,7 +6,6 @@
 
 #include <X11/Xlib.h>
 
-extern char *malloc(size_t);
 
 #ifdef HAS_FRAME_BUFFER
 void
--- acm-5.0/src/init.c.tim	Wed Jul  7 14:53:28 1999
+++ acm-5.0/src/init.c	Wed Jul  7 14:54:18 1999
@@ -70,14 +70,14 @@
 	char      libname[256];
 
 	if ((f = fopen(name, access)) == (FILE *) NULL) {
-		strcpy(libname, "../objects/");
+		strcpy(libname, "/usr/lib/games/acm/");
 		strcat(libname, name);
 		if ((f = fopen(libname, access)) == (FILE *) NULL) {
 			strcpy(libname, _acm_fopen_library_dir);
 			strcat(libname, name);
 			if ((f = fopen(libname, access)) == (FILE *) NULL) {
 				strcpy (libname, SRCDIR);
-				strcat (libname, "/../objects/");
+				strcat (libname, "/usr/lib/games/acm/");
 				strcat (libname, name);
 				if ((f = fopen(libname, access)) == (FILE *) NULL) {
 					fprintf(stderr, errmsg, name, _acm_fopen_library_dir, libname);
@@ -102,7 +102,7 @@
 
 	strcpy(pname, name);
 	if (stat(name, &statbuf) != 0) {
-		strcpy(pname, "../objects/");
+		strcpy(pname, "/usr/lib/games/acm/");
 		strcat(pname, name);
 		if (stat(pname, &statbuf) != 0) {
 			strcpy(pname, _acm_fopen_library_dir);
--- acm-5.0/dis/sdbm/sdbm.c.orig	Tue May 12 05:59:05 1998
+++ acm-5.0/dis/sdbm/sdbm.c	Mon Jul  7 13:36:17 2003
@@ -52,13 +52,6 @@
 /*
  * externals
  */
-#ifndef sun
-extern int errno;
-#endif
-
-extern char *malloc proto((unsigned int));
-extern void free proto((void *));
-extern long lseek();
 
 /*
  * forward
--- acm-5.0/dis/sdbm/util.c.orig	Tue Sep 26 19:02:06 1995
+++ acm-5.0/dis/sdbm/util.c	Mon Jul  7 13:38:25 2003
@@ -1,3 +1,4 @@
+#include <errno.h>
 #include <stdio.h>
 #ifdef SDBM
 #include "sdbm.h"
@@ -10,8 +11,7 @@
 register char *s1;
 register char *s2;
 {
-	extern int errno, sys_nerr;
-	extern char *sys_errlist[];
+	extern int sys_nerr;
 	extern char *progname;
 
 	if (progname)
